Dockerfile

```diff
@@ -12,6 +12,11 @@ RUN set -x; apk add --no-cache wget bzip2 ca-certificates && \
     chmod +x /usr/local/bin/restic && \
     mkdir /.cache && chmod -R 777 /.cache
 
+RUN set -x; apk add --no-cache wget bzip2 ca-certificates git gcc && \
```

Comment:
We should remove the previous `RUN` step. Loading restric is not necessary if we are going to build it on our own.

Reply:
Right, removed it :)